staging/src/k8s.io/kubectl/pkg/cmd/run/run.go

```diff
@@ -281,7 +281,7 @@ func (o *RunOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 		return cmdutil.UsageErrorf(cmd, "--port must be set when exposing a service")
 	}
 
-	namespace, _, err := f.ToRawKubeConfigLoader().Namespace()
+	namespace, enforceNamespace, err := f.ToRawKubeConfigLoader().Namespace()
```

Comment:
I know you are just following the pattern in this command, but it seems like most of the other commands have Namespace and EnforceNamespace as properties on their options struct.  And then do something like this:
https://github.com/kubernetes/kubernetes/blob/87e5d4e4dec69471a2d2e026174af239a6d73702/staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go#L275

So then you don't have to pass namespace and enforceNamespace to all the other methods as separate args.

Reply:
Great, I'll update this.